synfig-studio/src/gui/canvasview.cpp

```diff
@@ -2701,6 +2707,18 @@ CanvasView::set_onion_skins()
 	work_area->set_onion_skins(onion_skins);
 }
 
+void
+CanvasView::toggle_show_ruler()
+{
+	if(toggling_show_ruler)
```

Comment:
How this variable is used?

Reply:
Oh yes, thanks for pointing it out. I believe this was necessary when the button was still there to prevent the calling of the function again. I will remove it.